pages/sdk/protocol-kit/guides/safe-deployment/deploy-safe.mdx

```diff
@@ -0,0 +1,139 @@
+import { Steps } from 'nextra/components'
+
+# Deploy a Safe
+
+In this guide, you will learn how to deploy a new Safe using the Protocol Kit. This process includes initializing the Protocol Kit, setting up your Safe configuration, and executing the deployment.
+
+For more detailed information, see the [Protocol Kit Reference](../../../protocol-kit/reference/safe.mdx).
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Install dependencies
+
+First, you need to install the Protocol Kit.
+
+```bash
+pnpm add @safe-global/protocol-kit
+```
+
+## Steps
+
+<Steps>
+
+  ### Imports
+
+  Begin by importing the `Safe` class from the Protocol Kit.
```

Comment:
Perhaps using "begin" here and in the second step "firstly" is not super clean

Reply:
done!